web/src/user_group_edit.ts

```diff
@@ -1317,6 +1399,10 @@ export function setup_page(callback: () => void): void {
         );
         $groups_overlay_container.html(groups_overlay_html);
 
+        update_displayed_groups(
+            z.string().optional().parse(ls.get(user_group_visibility_key)) ??
```

Comment:
Can this directly use `dropdown_filters` or whatever the new name we decide for that variable?